jujugui/static/gui/src/app/components/added-services-list/test-added-services-list.js

```diff
@@ -103,35 +103,52 @@ describe('AddedServicesList', () => {
     assert.deepEqual(output, expected);
   });
 
-  it('performs the necessary work to focus a service', () => {
-    var allServices = [{
-      get: () => 'trusty/wordpress',
-      set: sinon.stub(),
-      setAttrs: sinon.stub()
-    }, {
-      get: () => 'trusty/apache',
-      set: sinon.stub(),
-      setAttrs: sinon.stub()
-    }, {
-      get: () => 'trusty/mysql',
-      set: sinon.stub(),
-      setAttrs: sinon.stub()
-    }];
-    var services = {
+  function createService(attrs) {
```

Comment:
Maybe you can combine this and the test util above and move it into the component test utils.